src/Microsoft.Health.Fhir.TemplateManagement/Models/ImageInfo.cs

```diff
@@ -55,35 +56,67 @@ public static bool IsDefaultTemplateImageReference(string imageReference)
         }
 
         public static bool IsValidImageReference(string imageReference)
+        {
+            try
+            {
+                ValidateImageReference(imageReference);
+            }
+            catch
+            {
+                return false;
+            }
+
+            return true;
+        }
+
+        public static void ValidateImageReference(string imageReference)
```

Comment:
Seems there is only one method calls ValidateImageReference, do we need to throw exceptions? Might be cleaner to just return false on fail? If we need the error message, I think we should throw an exception.